sys/src/cmd/map/libmap/simpleconic.c

```diff
@@ -11,33 +11,33 @@
 #include <libc.h>
 #include "map.h"
 
-static double r0, a;
+static double rr0, a;
```

Comment:
Out of interest, why was this renamed?  Did it clash with something?

Reply:
In libc.h r0 it's a function, since it's redeclared here, compiler didn't like. Simpler solution was renaming it.